### DISCUSSION TITLE
Sync Error

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#91] [Q&A]
While I am trying to sync the changes to the main repo, the following screenshot error appears that doesn't allow me to sync. 
Could you please help me to debug this error ? @eculler @nquarder 

Error: Pylance has crashed. Would you like to try using a Node.js executable to run the language server?

![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/38798140/d1f0b754-1705-4d88-9409-1d810e7637dd)

**Comment by eculler** [comment]
It looks like your sync went ok? Pylance does sometimes crash - I usually click cancel and it works out. As long as you see your sync in the repository, everything is fine.

**Reply by DharaniSureshBabu** [reply]
Thank you!
